python/ql/test/experimental/library-tests/frameworks/dill/Decoding.py

```diff
@@ -1,4 +1,4 @@
 import dill
 
-dill.loads(payload)  # $UNSAFE_getAnInput=payload $UNSAFE_getOutput=Attribute() $UNSAFE_getFormat=dill
-dill.loads(payload, encoding='latin1')  # $UNSAFE_getAnInput=payload $UNSAFE_getOutput=Attribute() $UNSAFE_getFormat=dill
+dill.loads(payload)  # $decodeInput=payload $decodeOutput=Attribute() $decodeFormat=dill $decodeUnsafe=
+dill.loads(payload, encoding='latin1')  # $decodeInput=payload $decodeOutput=Attribute() $decodeFormat=dill $decodeUnsafe=
```

Comment:
When there is no tag value, you can just use `$tag` instead of `$tag=`.

I also don't think we're gaining anything from having the example with `encoding` ? 

```suggestion
dill.loads(payload)  # $decodeInput=payload $decodeOutput=Attribute() $decodeFormat=dill $decodeUnsafe
```

Reply:
Not anymore when we are not extracting the encoding, no.